node/src/chain_spec.rs

```diff
@@ -118,6 +120,95 @@ pub fn development_config() -> Result<ChainSpec, String> {
 	))
 }
 
+pub fn lerna_config() -> Result<ChainSpec, String> {
+	let wasm_binary = WASM_BINARY.ok_or("Stakenet wasm binary not available".to_string())?;
+	let mut properties = Map::new();
+	properties.insert("tokenDecimals".into(), 12.into());
+	properties.insert("tokenSymbol".into(), "HDX".into());
+
+	Ok(ChainSpec::from_genesis(
+		// Name
+		"HydraDX Snakenet",
```

Comment:
shouldn't this be just "HydraDX" ? this shows up in js apps as chain name in top left